Sage/carbon
# docs: update testing documentation

### Proposed behaviour

Extend our testing documentation with new information about:
- dashboards;
- how to run `visual` tests;
- remove `chrome` browser form `GitHub actions` `yaml` file and from `.travis` file, we are using predefined `chrome` in `applitools`;
- add more parallel `jobs` for travis build;
- split buttonAllButtons for a 5 different feature files;
- add `chrome`, `firefox`,  `safari` per every check in `applitools`;
- add our maximum (as open source project) `concurrency` to 10.
### Current behaviour

We need to update testing documentation. 

### Checklist


- [ ] Screenshots are included in the PR

### Additional context


### Testing instructions
